tests/ui/imports/issue-121168-2.rs

```diff
@@ -0,0 +1,12 @@
+//@ edition: 2018
+//@ compile-flags: --extern issue_121168_extern
+//@ aux-build: issue-121168-extern.rs
```

Comment:
Could you make this a single test with `revisions` instead of having three tests?

Reply:
Fixed it.